File: 08.string/5622_dial.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string s;
    int sec=0;

    cin >> s;

    for (auto c : s) {
        if (c=='A' || c=='B' || c=='C')
            sec += 3;
        else if (c=='D' || c=='E' || c=='F')
            sec += 4;
        else if (c=='G' || c=='H' || c=='I')
            sec += 5;
        else if (c=='J' || c=='K' || c=='L')
            sec += 6;
        else if (c=='M' || c=='N' || c=='O')
            sec += 7;
        else if (c=='P' || c=='Q' || c=='R' || c=='S')
            sec += 8;
        else if (c=='T' || c=='U' || c=='V')
            sec += 9;
        else if (c=='W' || c=='X' || c=='Y' || c=='Z')
            sec += 10;
    }

    cout << sec << endl;

    return 0;
}

File: 08.string/5622_dial.py
#!/usr/bin/env python

s = input()

sec = 0
for c in s:
    if c in "ABC":
        sec += 3
    elif c in "DEF":
        sec += 4
    elif c in "GHI":
        sec += 5
    elif c in "JKL":
        sec += 6
    elif c in "MNO":
        sec += 7
    elif c in "PQRS":
        sec += 8
    elif c in "TUV":
        sec += 9
    elif c in "WXYZ":
        sec += 10

print(sec)
